smart-contracts/contracts/cl-vault/src/helpers.rs

```diff
@@ -312,4 +327,176 @@ mod tests {
         assert_eq!(round_up_to_nearest_multiple(-19, 5), -15);
         assert_eq!(round_up_to_nearest_multiple(-20, 5), -20);
     }
+
+    #[test]
+    fn test_get_single_sided_deposit_0_to_1_swap_amount() {
```

Comment:
Test cases for test_get_single_sided_deposit_0_to_1_swap_amount and test_get_single_sided_deposit_1_to_0_swap_amount funcs